TypeInjections/Tool/Tool.fs

```diff
@@ -36,15 +39,24 @@ module Tool =
           [<Option('c', "clone", Required = false, HelpText = "input a git URL")>]
           url: string
           [<Option('f', "force", Required = false, HelpText = "overwrite the Old and New folders in the Output folder")>]
-          force: bool }
+          force: bool
+          [<Option('i', "ignore", Required = false, HelpText = "specify filename(s) you want to ignore")>]
+          ignore: seq<string>}
+        
+    (* TypecartProject's 2nd argument can be:
+    1. the absolute path to .typecartignore file
+    2. a list of regex values *)
+    let handleIgnore(ignores: seq<string> ) : Regex list =
+        let li = ignores |> Seq.toList
+        List.map (fun (x:string)  -> x |> Regex) li 
 
     let checkInputs (paths: string list) =
         for a in paths do
             // ensure folders are in computer
             if Directory.Exists(a) = false then
                 failwith $"{a} is not found"
             // make sure folder has at least 1 dafny file
-            // (typeCart is already smart enough to ignore non-dafy files)
+            // (typeCart is already smart enough to ignore non-dafny files)
             let dafFiles = Directory.GetFiles(a, "*.dfy")
```

Comment:
Same here. You shouldn't do the filtering here. Typecart API can already detect which dafny files to read in.

Reply:
This is not filtering the files, this is an error checker function. I thought the error the `parseAST` function was misleading
```
error: list of files supplied to parser is empty
```
I wanted to give a more helpful error message that the folder path is real and there are files in the folder, but there aren't any Dafny files

Reply:
Then we should do the checking inside the API. Let me change that. I'm mildly against having to retrieve directory listing multiple times. The API should be designed in a way such that the user should never have to do any manual directory checking.

Reply:
I agree with what you're saying. Can I add a couple error checks to `parseAST` to help the user fix input issues?

Reply:
Removed this error check in `Tool`